tpch/distributed_queries/q01.sql

```diff
@@ -0,0 +1,23 @@
+-- using default substitutions
```

Comment:
we don't have to copy queries again - they are supposed to be the same as non-distributed queries

Reply:
changed back to the original table names